script.js

```diff
@@ -0,0 +1,13 @@
+const humburger = document.querySelector('.humburger');
+const navMenu = document.querySelector('.navbar-desk');
+
```

Comment:
- [ ] Great job. everything looks ok. lets perfect it by shifting the close button inside a little so it more visible
 - [ ] please shift the menu links down a little and make their font weight bolder . great job
 yours
![Screenshot (418)_LI](https://user-images.githubusercontent.com/65298048/188510943-5d378c8b-4e5c-4aeb-adb8-eb9841ef65db.jpg)
figma

![Screenshot (419)](https://user-images.githubusercontent.com/65298048/188510998-c5c18b74-a3e8-452b-a710-5a85baa4ca0e.png)

Reply:
Hi @tufoinnkuo10 
I added the eslintrc files but it is not checked anything yet kindly can you help me with this problem?